extensions/spatial.sigmf-ext.md

```diff
@@ -0,0 +1,362 @@
+The azimuth is specified in degrees east of true north and elevation (if
+provided) is in degrees above the horizon.
+
+The `emitter_bearing` field within a `captures` segment is used to specify the
+ground truth bearing of all signals contained within a multichannel dataset,
+relative to the `aperture_bearing`. This is useful for reference data which is
+well controlled, but is not well suited for arbitrary signals or data with more
+than one emitter location.
+
+The `phase_offset` field is a double precision value used when a dataset is
+captured from a RF device that is phase cohenert but not phase-aligned. Datasets
+making use of this field can be post-processed to align the data and this field
+can be set to zero. This value is always relative to channel 0, and therefore
+MUST be zero for channel 0. If this field is omitted then it is assumed that the
+value is zero, and thus it is always OPTIONAL for channel 0 or datasets that are
+already phase aligned.
+
+### 2.1 The `calibration` Object
+
+The `calibration` object is a special captures segment metadata field that
+indicates the segment is used for calibration. This might be used to show that
+a tone or broadboand noise signal was generated to perform phase alignment in
+post-processing. The resulting value from post-processing calibration can then
+be stored in the `captures` object `phase_offset` field .
+
+If this field is not defined for a `captures` segment, then that segment SHOULD
+be treated as normal data.
+
+|name|required|type|description|
+|----|--------|----|-----------|
+|`caltype`|true|[caltype](spatial.sigmf-ext.md#211-the-caltype-field)|Type of calibration data contained.|
+|`bearing`|false|bearing|The bearing of the calibration signal.|
+|`cal_geometry`|false|`cartesian_point`|The position of the calibration antenna phase center relative to the `spatial` CRS.|
+
+Either the `bearing` or the `cal_geometry` field MUST be provided if a captures
+segment includes the `calibration` field. The `bearing` object is best used to
+describe a remote calibration source location in a spherical coordinate system;
+the `cal_geometry` is best used when the calibration emitter is local to the
+multi-element array.
+
+#### 2.1.1 The `caltype` field
+
+The `caltype` field can have one of the following values:
+
+|value|description|
+|-----|-----------|
+|`tone`|This segment contains a tone for calibration purposes.|
+|`xcorr`|This segment contains a signal for crosscorrelation calibration purposes.|
+|`other`|This segment contains another type of calibration signal.|
+
+## 3 Annotations
+
+This extension adds the following optional fields to the `annotations` SigMF
+object:
+
+|name|required|type|description|
+|----|--------|----|-----------|
+|`signal_azimuth`|false|double|Azimuth in degrees east of north associated with the specific annotation.|
+|`signal_bearing`|false|`bearing`|Bearing associated with the specific annotation.|
```

Comment:
For the case of a collection, *which* of the `num_elements` `.sigmf-meta` files should `signal_{azimuth, bearing}` go in??  Please don't say "all of them" :-D

Surely this *doesn't* belong in the `.sigmf-collection` file ...

Some things are just *so* much easier in multi-channel recordings ...